Add information about the official bug bounty hotlist. (#68)
AndroidX/androidx: CONTRIBUTING.md

  - [Room](https://developer.android.com/topic/libraries/architecture/room)
  - [WorkManager](https://developer.android.com/topic/libraries/architecture/workmanager)

> For the list of bugs that you can contribute towards, please take a look at the official [bug bounty list](http://goo.gle/androidx-bug-bounty).

Reviewer: ```suggestion
Not sure where to start? Take a look at the official [bug bounty list](http://goo.gle/androidx-bug-bounty)!
```

Reviewer: Maybe something like this so that it doesn't seem like we won't accept contributions outside things we've already created bugs for?

Author: Done.

Reviewer 2: maybe call this bug/feature bounty list because most of them are actually features ?

Author: Done.
